scheduler/test/cook/test/tools.clj

```diff
@@ -791,6 +791,27 @@
       (is (= [(make-job 1 2 2048) (make-job 2 1 1024)]
              (util/filter-based-on-user-quota {test-user {:count 4, :cpus 20, :mem 6144}} user->usage queue))))))
 
+(deftest test-filter-pending-jobs-for-quota
+  (let [test-user-1 "john"
+        test-user-2 "bob"
+        user->usage {test-user-1 {:count 1, :cpus 1, :mem 1} test-user-2 {:count 1, :cpus 1, :mem 1}}
+        pool-quota {:count 4, :cpus 100, :mem 100}
+        job-user (atom 0)
+        make-job (fn [user]
+                   (swap! job-user inc)
+                   {:db/id @job-user
```

Comment:
Do we actually need the `:db/id` for this test? Seems like we could get rid of it and the `job-user` `atom`.

Reply:
Nope. Good find.